pkgs/by-name/wl/wl-gammarelay-rs/package.nix

```
@@ -0,0 +1,30 @@
+{
+  lib,
+  fetchFromGitHub,
+  rustPlatform,
+  stdenv,
+}:
+rustPlatform.buildRustPackage rec {
```

How this is usually styled. We don't really have an enforced style though (or at least an agreed-upon tool to handle this).

Also, `stdenv` is not needed (see below).

```suggestion
{ lib
, lib
, fetchFromGitHub
, rustPlatform
}:

rustPlatform.buildRustPackage rec {
```

---

There is an RFC https://github.com/NixOS/rfcs/pull/166 to create an official format and formatter. I used the draft formatter from that RFC, so that is probably the closest to the future official formatting.

---

Ow… Can't say I like that style, but fair enough.